app/build.gradle

```diff
@@ -32,4 +32,7 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test.ext:junit:1.1.0'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
+    testImplementation 'androidx.arch.core:core-testing:2.0.0'
+    androidTestImplementation 'androidx.arch.core:core-testing:2.0.0'
+    testImplementation 'io.mockk:mockk:1.9'
```

Comment:
This particular addition was to add the mockk in the build.gradle in order to access it within the test class. I also added some basic android test implementations for the @Test, etc.

Reply:
Good idea.